src/html2md.js

```diff
@@ -92,6 +97,12 @@ function addMetadata(hast, mdast) {
           meta.set(text(name), text(content));
         }
       }
+    } else if (child.tagName === 'script' && child.properties.type === 'application/ld+json') {
+      try {
+        meta.set(text('json-ld'), text(sanitizeJsonLd(toString(child))));
```

Comment:
why sanitize? those should be escaped by markdown to string

Reply:
ok, I think parsing to ensure the json is valid is useful at least